HW04/maksim_manzhai/spec/mentor_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'mentor'
+require 'student'
+require 'homework'
+require 'notification'
+
+RSpec.describe Mentor do
+  subject { described_class.new(name: 'Will', surname: 'Smith') }
+
+  let(:student) { Student.new(name: 'Chris', surname: 'Rock') }
+  let(:notification) { Notification.new(title: 'HW03', description: 'OOP in Ruby') }
+  let(:homework) { Homework.new(title: 'HW03', description: 'OOP in Ruby', student: student, mentor: subject) }
+
+  describe '#add_homework' do
+    before { subject.add_homework(title: 'HW03', description: 'OOP in Ruby', student: student) }
+
+    it 'returns an object of class Homework' do
+      expect(homework).to be_a(Homework)
+    end
+
+    it 'student gets notification about new homework' do
+      expect(student.notifications[0]).to be_a(Notification)
+    end
+  end
+
+  describe '#mark_as_read!' do
+    let(:notification) { Notification.new(title: homework.title, description: 'Homework started') }
+
+    before { subject.notifications << notification }
+
+    it 'gets notification' do
```

Comment:
1. you can test here and in **all similar cases** that the amount of notifications is changed 
2. express it in your it block